GameData/RP-1/Contracts/Earth Observation 1/EOSIRRad1.cfg

```diff
@@ -80,8 +80,9 @@ CONTRACT_TYPE
 				maxApA = 1000000
 				minPeA = 500000
 				maxEccentricity = 0.005
+				maxInclination = 45
 				disableOnStateChange = true
-				title = Achieve orbit between 500km and 1,000km in altitude with no more than .005 eccentricity.
+				title = Achieve orbit between 500km and 1,000km in altitude with no more than .005 eccentricity and inclination lower than 50 degrees.

```

Comment:
I don't think I'd suggest having a max inclination here? It makes it basically impossible to complete when launching from high latitudes. I think it's fine for later contracts to have _minimum_ inclination requirements, but a max requirement seems off to me.

Also the number and the text don't match. ;)

Reply:
Ahh, right, the whole not everyone launches from the Cape bit.